vevestaX/vevesta.py

```diff
@@ -201,91 +204,87 @@ def dump(self, techniqueUsed, filename=None, message=None, version=None, showMes
             df_messages.to_excel(writer, sheet_name='messages', index=False)
             pandas.DataFrame(self._data).to_excel(writer,sheet_name='sampledata',index=False)  
 
-        # self._plot(filename)
+        # self.__plot(filename)
 
         if showMessage:
-            message = self.getMessage()
+            message = self.__getMessage()
             print(message)
 
-        
-
-
 
-    def _plot(self, filename):
+    def __plot(self, fileName):
 
         modelingData = None
 
-        if (filename == None):
+        if (fileName == None):
             return print("Error: Provide the Excel File to plot the models")
 
-        # skipcols = ['experimentID', 'features']
-        # cols = for column in modelingData
-
-        if (os.path.isfile(filename)):
-            modelingData = pandas.read_excel(filename, sheet_name='modelling', index_col=[])
-
-        modelingData['timestamp in UTC'] = pandas.to_datetime(['timestamp in UTC'],)
-        print(modelingData['timestamp in UTC'].dtypes)
-
+        if (os.path.isfile(fileName)):
```

Comment:
you can write this whole logic in a function, __getModellingDataFromExcel(filename).

Reply:
done